### ISSUE TITLE
how to set the alpha parameters?

### ISSUE THREAD

**Issue by ray-lee-94** [clovaai/overhaul-distillation#2]

**Comment by bhheo** [comment]
Hello,
The alpha parameter is included in loss summation.

- CIFAR-100/train_with_distillation.py (line 91)
`loss = loss_CE + loss_distill.sum() / batch_size / 1000`
The alpha is 0.001

- ImageNet/train_with_distillation.py (line 179)
`loss = loss_CE + loss_distill.sum() / batch_size / 10000`
The alpha is 0.0001

The alpha values of code are same as the paper.
If you want to change the alpha value, change the corresponding lines.

**Comment by ray-lee-94** [comment]
Thank you for your replay. I do read your code and paper and try to transfer on my dataset.
So i want to know the ruler to set alpha.  Besides that, I try to distillate from the pretrained darknet53. 
The structure of darknet53 is differernt from resnet as the DarkUnit has relu before the skip connection. This structure breaks your preReLU setting. Any suggestion would be greatfull.

**Comment by bhheo** [comment]
The alpha was determined by grid search on a 10 log scale.
It depends on the task of network and does not depend on the network.

So, I recommend you to set alpha to
10^-3 for 32x32 image
10^-4 for 224x224 scale classification task
10^-5 for detection and segmentation task

I have never considered darknet for distillation, and it seems difficult to apply preReLU position.

I think preReLU position at second conv-bn-relu in residual block can be an option.
And it must be last residual block before pooling or stride conv layer.
But I can't guarantee it will work.

**Comment by ray-lee-94** [comment]
Thank you very much. I set the preReLU after the first stride conv in each stage. But the result is far worse than not using distillation. I will try your idea and report the result. 
I have another question:
        since darknet adopted special LeakyRelu which has a pass for the negtive information.  Should i make some modification on the distill loss function or something else?
